### PR TITLE
Feat create post and explore

### PR DESCRIPTION
**Pull request by SinchanaMS**
This request adds the following features:

1. Create Post - with text and image
2. Sidebar - Navigation and Followers Suggestions
3. Explore Feed

## Preview
image

### CODE REVIEW

File: src/components/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "react-router-dom";
+import { AiFillHome, AiFillCompass, AiFillBell } from "react-icons/ai";
+import { MdOutlineLogout } from "react-icons/md";
+import { BsFillBookmarkFill } from "react-icons/bs";
+
+export default function Sidebar() {
+  return (
+    <div className="flex-none flex flex-col gap-7 text-xl bg-gray-50 h-[80vh] py-4 px-5  my-4 mx-3 shadow-md rounded-md lg:pr-12">
+      <NavLink to="/" className="flex gap-3 items-center">
+        <AiFillHome />
```
Comment (theVivek-kumar) [inline src/components/Sidebar.jsx:10]: Ui looking  amazing  good to merge  unable to find any improvement

**Review by theVivek-kumar — APPROVED** [review]

**Comment by srejitk** [comment]
Good work. No improvements suggested.

File: src/components/Post.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                  : "http://cdn.onlinewebfonts.com/svg/img_264570.png"
+              }
+              alt="user-dp"
+              className="h-9 w-9 rounded-full aspect-square "
+            />
+
+            <div className="flex items-center justify-between w-full">
+              <div>
+                <p className="text-sm font-semibold">{post.author}</p>
+              </div>
```
Comment (pankaj-prog) [inline src/components/Post.jsx]: Is this div really needed here

**Review by pankaj-prog — APPROVED** [review]
Suggestions : 
- I see multiple div's in the post component, it's better if you could use semantics 
- Also you could use absolute imports, they are a much cleaner way to import component and other things.

File: src/components/Post.jsx (hunk 2 above)
Comment (SinchanaMS) [reply src/components/Post.jsx]: Oh! I had it to include another field under the author which I later removed. 
I have removed it now and edited the code to use semantic tags as well, thank you!

File: src/components/Sidebar.jsx (hunk 1 above)
Comment (SinchanaMS) [reply src/components/Sidebar.jsx:10]: Thank you for the review!

**Comment by SinchanaMS** [comment]
Thank you for the reviews, made the necessary changes. Merging to dev.

**Pull request merged**